src/SFA.DAS.EmployerAccounts.Web.UnitTests/Orchestrators/EmployerAccountOrchestratorTests/WhenCreatingTheUserAccount.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Web;
+using MediatR;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EmployerAccounts.Commands.CreateUserAccount;
+using SFA.DAS.EmployerAccounts.Configuration;
+using SFA.DAS.EmployerAccounts.Interfaces;
+using SFA.DAS.EmployerAccounts.Models.Account;
+using SFA.DAS.EmployerAccounts.Web.Orchestrators;
+using SFA.DAS.EmployerAccounts.Web.ViewModels;
+using SFA.DAS.NLog.Logger;
+
+namespace SFA.DAS.EmployerAccounts.Web.UnitTests.Orchestrators.EmployerAccountOrchestratorTests
+{
+    public class WhenCreatingTheUserAccount
+    {
+        private EmployerAccountOrchestrator _employerAccountOrchestrator;
+        private Mock<IMediator> _mediator;
+        private Mock<ILog> _logger;
+        private Mock<ICookieStorageService<EmployerAccountData>> _cookieService;
+        private EmployerAccountsConfiguration _configuration;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _mediator = new Mock<IMediator>();
+            _logger = new Mock<ILog>();
+            _cookieService = new Mock<ICookieStorageService<EmployerAccountData>>();
+            _configuration = new EmployerAccountsConfiguration();
+
+            _employerAccountOrchestrator = new EmployerAccountOrchestrator(
+                _mediator.Object, 
+                _logger.Object,
+                _cookieService.Object, 
+                _configuration);
+
+            _mediator.Setup(x => x.SendAsync(It.IsAny<CreateUserAccountCommand>()))
+                .ReturnsAsync(new CreateUserAccountCommandResponse()
+                {
+                    HashedAccountId = "ABS10"
+                });
+        }
+
+        [Test]
+        public async Task ThenTheMediatorCommandIsCalledWithCorrectParameters()
```

Comment:
Can we make the test names less technical?  They're describing the implementation which isn't ideal.

(I realise you copied these from somewhere else! :))